orchestra/analytics/latency.py

```diff
@@ -128,7 +135,14 @@ def project_time_row_generator(project_information,
                     assignment['worker'],
                     timedelta(seconds=snapshot['work_time_seconds']),
                     parse(snapshot['datetime'])))
-
+            if in_progress_iterations and assignment['status'] == 'Processing':
+                iterations.append(Iteration(
```

Comment:
it looks like there are three ways to append to iterations atm---can you describe all three cases?